<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurstSearchWidget</class>
 <widget class="QWidget" name="BurstSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Burst Search Method (&amp;single channel)</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxMethod</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxMethod">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Sliding</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lee filter + IPT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Binned</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>203</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonBurstView">
     <property name="text">
      <string>Show average alternation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxMore">
     <property name="text">
      <string>More Options</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>&amp;perform Burst Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Burst Search Method (&amp;both channels)</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxMethodDual</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxMethodDual">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>All Photons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dual Channel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Combined</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxBSParams">
     <property name="title">
      <string>Burst Search Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="4" column="0" colspan="3">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Maximal Burst &amp;Rate</string>
           </property>
           <property name="buddy">
            <cstring>spinBoxMaxRateGlobal</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Minimal Burst &amp;Rate</string>
           </property>
           <property name="buddy">
            <cstring>spinBoxMinRateGlobal</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBoxMinRateGlobal">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string>kHz</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBoxMaxRateGlobal">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string>kHz</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxMinSizeGlobal">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Global Burst Parameters</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Minimal Burst &amp;Size</string>
           </property>
           <property name="buddy">
            <cstring>spinBoxMinSizeGlobal</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2" rowspan="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QFrame" name="frameLee">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&amp;Window size</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_LeeWin</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;Variance</string>
           </property>
           <property name="buddy">
            <cstring>doubleSpinBox_LeeVar</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Minimal &amp;inter
photon time</string>
           </property>
           <property name="buddy">
            <cstring>doubleSpinBox_IPT</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinBox_LeeWin">
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_LeeVar">
           <property name="suffix">
            <string/>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_IPT">
           <property name="suffix">
            <string>us</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSpinBox" name="spinBox_LeeWin_2">
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_LeeVar_2">
           <property name="suffix">
            <string/>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_IPT_2">
           <property name="suffix">
            <string>us</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frameSliding">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QComboBoxChannel" name="comboBoxChannel1"/>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="spinBox_L">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_M">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Minimal
Burst &amp;Size</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_L</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Time
window &amp;T</string>
           </property>
           <property name="buddy">
            <cstring>doubleSpinBox_T</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_T">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string>us</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_T_2">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string>us</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spinBox_L_2">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_M_2">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Number of
neighbouring
photons &amp;M</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_M</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBoxChannel" name="comboBoxChannel2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Channel</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxChannel1</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>LeeFilter + IPT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Sliding Burst Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5" rowspan="3">
    <widget class="QGroupBox" name="groupBoxInfo">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowserParams">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Burst Search&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Parameters Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTextBrowser" name="textBrowserResult">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Burst Search&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Result Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxSaveDual">
        <property name="text">
         <string>Save Dual Bursts</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxSaveSingle">
        <property name="text">
         <string>Save Single Bursts</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>compare Burst Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxMoreOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ini File</string>
        </property>
        <property name="buddy">
         <cstring>lineEditIni</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QEnhancedLineEdit" name="lineEditIni"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonSaveIni">
        <property name="text">
         <string>Save Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" colspan="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qenhancedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBoxChannel</class>
   <extends>QComboBox</extends>
   <header>../alexplots.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxMethod</tabstop>
  <tabstop>spinBox_M</tabstop>
  <tabstop>spinBox_M_2</tabstop>
  <tabstop>spinBox_L</tabstop>
  <tabstop>spinBox_L_2</tabstop>
  <tabstop>doubleSpinBox_T</tabstop>
  <tabstop>doubleSpinBox_T_2</tabstop>
  <tabstop>spinBox_LeeWin</tabstop>
  <tabstop>spinBox_LeeWin_2</tabstop>
  <tabstop>doubleSpinBox_LeeVar</tabstop>
  <tabstop>doubleSpinBox_LeeVar_2</tabstop>
  <tabstop>doubleSpinBox_IPT</tabstop>
  <tabstop>doubleSpinBox_IPT_2</tabstop>
  <tabstop>comboBoxMethodDual</tabstop>
  <tabstop>textBrowserParams</tabstop>
  <tabstop>textBrowserResult</tabstop>
  <tabstop>lineEditIni</tabstop>
  <tabstop>pushButtonSaveIni</tabstop>
  <tabstop>checkBoxSaveSingle</tabstop>
  <tabstop>checkBoxSaveDual</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxMore</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxMoreOptions</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMore</sender>
   <signal>clicked(bool)</signal>
   <receiver>frame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
